src/util/client/mod.rs

```diff
@@ -65,7 +66,6 @@ pub trait Client {
         log::debug!("Eval get response: {:#?}", response);
         match response.status() {
             StatusCode::OK => Ok(Some(response.json().await?)),
-            StatusCode::NOT_FOUND => Ok(None),
```

Comment:
I think we still need this, as without this, we would never get `Ok(None)`, which is a valid response and not an error.

Reply:
Good catch ! I accidentally deleted that